charts/vela-core/values.yaml

```diff
@@ -18,7 +18,7 @@ concurrentReconciles: 4
 
 ## @param controllerArgs.reSyncPeriod The period for resync the applications
 controllerArgs:
-  reSyncPeriod: 5m
+  reSyncPeriod: 3m
```

Comment:
let's seprate the state keep change in a new PR along with the belowing kubeClient QPS

Reply:
Okay